mapbox_tilesets/utils.py

```diff
@@ -126,9 +117,12 @@ def validate_geojson(count, feature):
             },
         },
     }
+    validate(instance=feature, schema=schema)
+    geojsonFeature = geojson.loads(json.dumps(feature))
+    if not geojsonFeature.is_valid:
+        raise mapbox_tilesets.errors.TilesetsError(f"Error in feature number {index}: " + ''.join(geojsonFeature.errors()))
 
-    validate_linear_ring(count, feature["geometry"])
-    return validate(instance=feature, schema=schema)
+    return
```

Comment:
this makes more sense to return nothing because this function `validate_geojson` is called 3 times, and the result is never used. `validate` either throws an error or does not, but never actually returns anything